test/js/lib/mlang.js

```diff
@@ -0,0 +1,496 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+require('../../../src/lib/dao');
+require('../../../src/lib/mlang');
+
+function buildPersonDAO() {
```

Comment:
When I wrote that I expected it to go in multiple different blocks. That didn't actually happen, so I've pulled this into the main describe's `beforeEach`.